<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMusicInfo_Window</class>
 <widget class="QMainWindow" name="MyMusicInfo_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMusicInfo</string>
  </property>
  <widget class="QWidget" name="awidget">
   <widget class="QLabel" name="windowTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Music - Create/Update Database</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="dbActions_Group">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Database Actions</string>
    </property>
    <widget class="QRadioButton" name="replaceData_Radio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Replace ALL Data</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="addData_Radio">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>105</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="selectDB_Group">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Database</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>285</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="production_Radio">
        <property name="text">
         <string>Production/Falcon</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="test_Radio">
        <property name="text">
         <string>Test/localhost</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>89</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressBar_Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QLabel" name="messages_Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="messages_List">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>601</width>
      <height>251</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Music Directory</string>
    </property>
   </widget>
   <widget class="QComboBox" name="musicDir_Box">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>89</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
